text_area.go

```diff
@@ -142,6 +169,41 @@ func (self *TextArea) atLineStart() bool {
 		(len(self.content) > self.cursor-1 && self.content[self.cursor-1] == '\n')
 }
 
+func (self *TextArea) atLineEnd() bool {
+	return self.atEnd() ||
+		(len(self.content) > self.cursor && self.content[self.cursor] == '\n')
+}
+
+func (self *TextArea) BackSpaceWord() {
+	if self.atLineStart() {
+		self.BackSpaceChar()
+		self.clipboard = "\n"
```

Comment:
is using a newline character what vim typically does?

Reply:
Ah, clipboard is kept.